components/devtools/actors/console.rs

```diff
@@ -266,14 +252,12 @@ impl ConsoleActor {
             .or_default()
             .push(CachedConsoleMessage::PageError(page_error.clone()));
         if id == self.current_unique_id(registry) {
-            let msg = PageErrorMsg {
-                from: self.name(),
-                type_: "pageError".to_owned(),
-                page_error,
+            match &self.root {
+                Root::BrowsingContext(bc) => registry
+                    .find::<BrowsingContextActor>(bc)
+                    .page_error(page_error),
+                Root::DedicatedWorker(_) => (),
```

Comment:
Yes, the motivation behind having both was because the error could come from a worker as well, but it's better as you say since we have no support for them yet in the devtools